Sources/Segment/Analytics.swift

```diff
@@ -111,6 +111,15 @@ extension Analytics {
         return settings
     }
     
+    /// Manually enable a destination plugin.  This is useful when a given DestinationPlugin doesn't have any Segment tie-ins at all.
+    /// This will allow the destination to be processed in the same way within this library.
+    /// - Parameters:
+    ///   - plugin: The destination plugin to enable.
+    public func manuallyEnableDestination(plugin: DestinationPlugin) {
```

Comment:
I'm going back and forth if we want this to be a "core" public API. It makes sense for this to be separated by an extension to me. Thoughts?